code/modules/mob/living/simple_mob/subtypes/mechanical/disbot_vr.dm

```diff
@@ -0,0 +1,50 @@
+/mob/living/simple_mob/mechanical/infectionbot
+	name = "Strange robot"
+	desc = "You get the feeling you should run."
+	icon = 'icons/mob/vagrant_vr.dmi'
+	icon_state = "vagrant"
+	icon_living = "vagrant"
+	icon_dead = "vagrant"
+	icon_gib = "vagrant"
+
+	maxHealth = 65
+	health = 40
+	movement_cooldown = 4
+
+	response_help = "pets the"
+	response_disarm = "gently pushes aside the"
+	response_harm = "hits the"
+	faction = "vagrant"
+	harm_intent_damage = 3
+	melee_damage_lower = 6
+	melee_damage_upper = 9
+	light_color = "#8a0707"
+	attacktext = "mauled"
+	attack_sound = 'sound/weapons/bite.ogg'
+
+	ai_holder_type = /datum/ai_holder/simple_mob/melee
+	say_list_type = /datum/say_list/disbot
+
+	var/poison_chance = 100
+	var/poison_per_bite = 5
+	var/poison_type = "zombiepowder"
+
+/datum/say_list/disbot
+	speak = list("ATTEMPTING TO CONTACT A.R.K, ATTEMPT 1e26+3","DIRT SAMPLE COLLECTED, DIRT QUOTA 124871/155 CONFIRMED.")
+	emote_see = list("Scans the dirt around it","Beeps as it scans a rock nearby")

```

Comment:
do not forget that emotes will be written as 
"Strange robot Scans the dirt around it" and such, so in emotes first letter needs to capitalization

Reply:
fixd